create-neon/src/shell.ts

```diff
@@ -15,19 +16,31 @@ export default function shell(cmd: string, args: string[], cwd: string): Promise
     reject = rej;
   });
 
-  child.on('exit', (code) => {
+  child.on("exit", async (code) => {
     if (code == null) {
-      process.exit(1);
-    }
+	reject(Error(`error code: ${code}`))
+    };
     if (code !== 0) {
-      process.exit(code);
-    }
+	reject(Error(`error code: ${code}`))
+    };
+
+    if (code === 0) {
+      try {
+          let data = await fs.readFile(path.join(cwd,'package.json'),'utf8')
+          let { description, author, license } = JSON.parse(data);
+          if ([description, author, license].includes(undefined)) {
```

Comment:
This could benefit from a comment explaining that this is testing to ensure `npm init` was successful. It will catch `Ctrl+C`, answering `no` and many other failures.

Reply:
See new [comments](https://github.com/neon-bindings/neon/commit/f7bb6e540301ce26638960dccf298e49650a9eb9)